### PR TITLE
Improve survey points

### PR DESCRIPTION
**Pull request by tudorbarascu**
* added the fk_object_reference column
* created geometry index
* added a default sequence on id
* added en + ro translation of chevillère (I guess it's this.. right? https://en.wikipedia.org/wiki/Tape_measure#The_measuring_tapes)
* added Total Station as a type of survey points ( I guess you are also using Total Stations, right? )

I haven't created the delta file etc. yet as the automated system is not working currently and there are the others pull requests that may be merged or not. I will do it as soon as everything is clear.

Also, the ``COMMENT ON TABLE qwat_od.surveypoint IS 'Table for sampling points. Inherits from surveypoint.';`` is obsolete, right?

### CODE REVIEW

File: value_lists/vl_survey_type.sql [hunk 1]
```diff
@@ -18,5 +18,6 @@ INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (102,
 INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (103, 'to be determined', 'à déterminer', 'de determinat');
 
 INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (9901, 'GPS'      , 'GPS'       , 'GPS' );
-INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (9902, 'meter?', 'chevillère'     , '?');
+INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (9902, 'measuring tape', 'chevillère'     , 'ruletă');
```
Comment (lbartoletti) [inline value_lists/vl_survey_type.sql]: Personnaly, I prefer "ruban métrique", "mètre à ruban" or simply **"mètre"** for the french translation.

File: value_lists/vl_survey_type.sql [hunk 2]
```diff
@@ -18,5 +18,6 @@ INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (102,
...
+INSERT INTO qwat_vl.survey_type (id, value_en, value_fr, value_ro ) VALUES (9903, 'total station', 'station totale'     , 'stație totală');
```
Comment (lbartoletti) [inline value_lists/vl_survey_type.sql]: 👍

**Comment by tudorbarascu** [comment]
@lbartoletti Should be done now! Also with ``ruban métrique``
Thanks for reviewing.

**Comment by haubourg** [comment]
> What do you think about adding a default squence on qwat_od.surveypoint (id)?

Yes, let's do that in another PR.

**Pull request merged**